requirements/README.rst

```diff
@@ -12,22 +12,27 @@ directly in the requirements directory.)
 
 .. _OEP-18: https://github.com/openedx/open-edx-proposals/blob/master/oeps/oep-0018-bp-python-dependencies.rst
 
-Upgrading all dependencies
-**************************
+While the ``*.in`` files are intended to be updated manually, the updating the ``*.txt`` files should only be manipulated using Makefile targets in a Linux environment (to match our build and deploy systems). Developers on Mac should prefer to use the GitHub workflows or run Make targets from inside devstack's lms-shell or another Linux environment.
```

Comment:
1. typo fix.
2. Reworded the second sentence to more directly tie it to the former statement.
```suggestion
While the ``*.in`` files are intended to be updated manually, updating the ``*.txt`` files should only be manipulated using Makefile targets in a Linux environment (to match our build and deploy systems). For developers on Mac, this can be achieved by using the GitHub workflows or by running Make targets from inside devstack's lms-shell or another Linux environment.
```